node_modules/col-canvas/lib/poller.js

```diff
@@ -482,8 +483,7 @@ var handleAssignment = function(ctx, users, categories, opts, assignment, callba
 };
 
 /**
- * Handle the category synchronisation for an assignment. If a new category
- * gets creates it will get added to the set of categories
+ * Handle the category synchronisation for an assignment, creating a new category if necessary
```

Comment:
@paulkerschen I like that you preserved the British spelling of "synchronisation".